distribution/docs/src/main/resources/_contents/_endpoints/csw-endpoint-contents.adoc

```diff
@@ -1230,6 +1230,43 @@ It is possible to receive a response to a `GetRecords` query that conforms to th
 </csw:GetRecordsResponse>
 ----
 
+.`GetRecords` XML Request using UTM Coordinates
+UTM coordinates can be used when making a CSW GetRecords request using an `ogc:Filter`.
+UTM coordinates should use EPSG:326XX where XX is the zone within the northern hemisphere.
```

Comment:
I think we want to be more explicit here. The `EPSG:326XX` is the srsName.  Do we also require the exact urn that is in the sample below? Just want to remove any ambiguities.